nni/common/concrete_trace_utils/concrete_tracer.py

```diff
@@ -1461,10 +1512,50 @@ def f(x, y):
         else:
             assert node_a.op == node_b.op and target_a == target_b
 
+    # apply dead code elimination according to the switch param `dce`
+    # note that we perform dce before instantiating GraphModule
+    # since there maybe some problems if we use GraphModule.recompile()
+    dont_delete = [operator.setitem, builtins.next]
+    if dce:
```

Comment:
is this the same as
https://github.com/pytorch/pytorch/blob/e9a7115605ee5b6ae38ea5716a4abea5aa415333/torch/fx/graph.py#L1375